tests/integration/targets/iosxr_config/tests/cli/prefix_set_remark.yaml

```diff
@@ -10,15 +10,15 @@
   register: result
   cisco.iosxr.iosxr_config:
     lines:
-      - "＃#﹟testremark"
```

Comment:
what is being removed here? looks like a # to me? but tiny?

Reply:
yes